package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "@types/bluebird": "^3.5.35",
-    "@types/node": "^15.3.1",
+    "@types/node": "^20.3.1",
```

Comment:
Out of curiosity - this bump fixed anything?
20.3.1 was supposed to be targeting Nodes >= 4.3 we have 4.2 so 18.15.3 should be enough?